daemon/main.go

```diff
@@ -138,6 +138,11 @@ func init() {
 						Name:        "keep-config",
 						Usage:       "When restoring state, keeps containers' configuration in place",
 					},
+					cli.BoolFlag{
+						Destination: &config.KVStoreIPv4Registration,
```

Comment:
We don't have to do this in this PR but we can add an identity to each node using a key or certificate and associate the lookup with the identity so a node can reclaim its own node address.